ir/value.h

```diff
@@ -42,6 +42,8 @@ class Value {
 
   static void reset_gbl_id();
 
+  virtual bool isIntConst(int64_t &n) const { return false; }
```

Comment:
this is not needed.

Reply:
i added this to be able to retrieve the value of the constant in `BinOp::getTypeConstraints()`'s `::ExtractValue` switch case. I couldn't find any easy way to retrieve this constant from a `Value`. Do you have some suggestions?